native/v-kit.sh

```diff
@@ -80,8 +80,21 @@ install_mezo() {
 }
 
 install_skip() {
```

Comment:
Given the existing problems with the official script and latest tag, I think we should default to our custom script installing a fixed version of Connect, e.g. 2.1.2 (let's use the same as Docker/Helm). Thoughts?

Reply:
Yeah, no problem ;)

Reply:
Addressed!